__tests__/app.test.js

```diff
@@ -78,3 +78,38 @@ describe.only("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: should return the array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(13);
+        body.forEach((articles) => {
+          expect(articles).toMatchObject({
+            author: expect.any(String),
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+        console.log(body);
```

Comment:
are there any keys you're expecting to not show up? For example, `body`? Think about the BBC website - you don't receive any of the article body when you're browsing articles, so it's not necessary to fetch that data until someone clicks onto a specific article (otherwise you're fetching data you won't use). This is something we need to test.